src/aap_eda/wsapi/messages.py

```diff
@@ -46,6 +46,7 @@ class ActionMessage(Message):
     delay: Optional[float]
     message: Optional[str]
     kind: Optional[str]
+    controller_job_id: Optional[str]
```

Comment:
What's the difference between this new `controller_job_id` and the existing `job_id`?

Reply:
@Alex-Izquierdo I thought they are the same but it predates us and is a unique job identifier for every action that we run. Initially I thought it is the job id from controller only to realize its a UUID and not the controller job id

Reply:
Thanks for the clarification, but the question was made by @hsong-rh xD